src/OpenLoco/src/Objects/VehicleObject.h

```diff
@@ -57,24 +57,23 @@ namespace OpenLoco
     };
     static_assert(sizeof(VehicleObjectFrictionSound) == 0xB);
 
-    struct VehicleObjectEngine1Sound
+    struct VehicleSimpleMotorSound
     {
         uint8_t soundObjectId;     // 0x0
         uint16_t defaultFrequency; // 0x1
         uint8_t defaultVolume;     // 0x3
-        uint16_t var_04;
-        uint8_t var_06;
-        uint16_t var_07;
-        uint8_t var_09;
+        uint8_t pad_04[3];
```

Comment:
these are not padding

Reply:
I'm pretty sure we have implemented all the sound code at this point. Unsure why they are var